libtuf-server/src/main/scala/com/advancedtelematic/libtuf_server/http/ServiceHttpClient.scala

```diff
@@ -18,15 +18,15 @@ import cats.syntax.option._
 import scala.concurrent.Future
 import scala.reflect.ClassTag
 
-
+@deprecated("use data type from libats-http", "v0.4.0-40")
 trait ServiceHttpClientSupport {
   def defaultHttpClient(implicit system: ActorSystem, mat: Materializer): (HttpRequest => Future[HttpResponse]) = {
     val _http = Http()
     req => _http.singleRequest(req)
   }
 }
 
-// TODO: Move to libats?
+@deprecated("use data type from libats-http", "v0.4.0-40")
 abstract class ServiceHttpClient(httpClient: HttpRequest => Future[HttpResponse])
```

Comment:
Why don't you use the client from `libats-http v0.4.0-40` ?

Reply:
Oops.